doc/source/dev_guide.rst

```diff
@@ -11,23 +11,105 @@ This guide provides essential information for developers contributing to NeuralO
+    depends on `PyTorch <https://pytorch.org>`_. As of spring 2025, PyTorch has stopped releasing updates to the Anaconda
+    distribution. For the latest PyTorch we recommend you use ``pip``. 
+
+Create your virtual environment and store it in the top level:
+
+.. code-block:: bash
+
+    python -m venv .venv
+    source .venv/bin/activate
+
+Install PyTorch. These are generic instructions; if you require a specific build, or a specific CUDA version, your installation
+command will vary. Check PyTorch's `getting started page <https://pytorch.org/get-started/locally/>`_ for more detailed instructions
+
+**For CUDA installations:**
+
+.. code-block:: bash
+
+    pip install torch torchvision --index-url https://download.pytorch.org/whl/<YOUR_CUDA_VERSION>
+
+**For CPU installations:**
+
+.. code-block:: bash
+
+    pip install torch torchvision --index-url https://download.pytorch.org/whl/cpu
+
+Next, install the library locally in editable mode, along with the ``".[dev]"`` tag to install all development dependencies 
+
+.. code-block:: bash
+    
     pip install -e .[dev]
 
+To test your installation, run python in interactive mode and import the library as ``neuralop`` to ensure it is properly built:
+
+.. code-block:: bash
+
+    $ python
+    Python 3.10.14 (main, Month Day Year, Time of Day) [GCC VERSION] on linux
+    Type "help", "copyright", "credits" or "license" for more information.
+    >>> import neuralop
+    >>> 
+
+You have now successfully built ``neuralop``. For instructions on contributing code, see below. 
+
 ~~~~~~~~
 
-Code Style and Standards
-------------------------
+Contributing code
+-----------------
+
+We welcome new contributions to the library! Our mission for NeuralOperator is to provide access
+to well-documented, robust implementations of neural operator methods from foundations to the cutting edge. 
+The library is primarily intended for methods that directly relate to operator learning: new architectures, 
+meta-algorithms, training methods and benchmark datasets. We are also interested in integrating interactive
+examples that showcase operator learning in action on small sample problems. 
+
+If your work provides one of the above, we would be thrilled to integrate it into the library. Otherwise, if your
+work simply relies on a version of the NeuralOperator codebase, we recommend publishing your code separately using 
+a procedure outlined :ref:`here <publishing_code_built_on_neuralop>`.
+
+Extending NeuralOperator
+++++++++++++++++++++++++
+
+To add a new neural operator model:
+
+1. Create a new file in ``neuralop/models/``.
+2. Implement the model as a subclass of ``BaseModel``.
+3. Add a parametrized unit test file in ``neuralop/models/tests``.
+
+To add a layer:
+
+1. Create a new file in ``neuralop/layers``
+2. Ensure the layer is a subclass of ``torch.nn.Module``.
+3. Add a parametrized unit test file in ``neuralop/layers/tests``.
+
+.. note ::
```

Comment:
I think you need a free line after this and no space between note and the colons - can you check all the syntax is correct and the document is rendered correctly?

Reply:
Will do.

Reply:
This builds properly